test/utils/unit-tests-data.js

```diff
@@ -25,9 +25,10 @@ module.exports = {
         content_encoding: 'gzip',
         content_language: 'en-US',
         metadata: { 'some-key': 'some-value' },
-        cache_control: 'max-age=60'
+        cache_control: 'max-age=60',
+        sse_mks_key_id: "sse-key-arn"
```

Comment:
You should change this to single quotes for consistency with the rest of the code.